src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,60 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's address block in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidBlock(String)}
+ */
+
+public class Block {
+
+    public static final String EXAMPLE = "311";
+    public static final String MESSAGE_BLOCK_CONSTRAINTS = "Address value should contain only numbers";
```

Comment:
`"Address value should contain only numbers"` should also update the sentence right?